src/pages/check-answers/template.njk

```diff
@@ -21,12 +21,29 @@
       actions: {
         items: [
           {
-            href: "#",
+            href: "/demo",
             text: "Change",
             visuallyHiddenText: "name"
           }
         ]
       }
+    },
+    {
+      key: {
+      text: "Telling Civil Service Pension Scheme"
+    },
+      value: {
+      text: mapping['tell-civil-service-pension'][session["tell-civil-service-pension"]]
+    },
+      actions: {
+      items: [
+        {
+          href: "/civil-service-pension",
+          text: "Change",
+          visuallyHiddenText: "name"
```

Comment:
This is probably not quite right

Reply:
What does that mean?  It renders the response but are you saying we could do it more elegantly?